src/libs/CategoryOptionListUtils.ts

```diff
@@ -0,0 +1,282 @@
+// eslint-disable-next-line you-dont-need-lodash-underscore/get
```

Comment:
why do we need lodash get here?

Reply:
to be fair, idk, this is copied over from the previously existing code. I think the lodashGet function simplifies a rather complicated nested lookup for us here (we'd need to write our own helper function for this case)